### ISSUE TITLE
razorpay not working in iOS using Xcode 10

### ISSUE THREAD

**Issue by ravifullestop** [razorpay/react-native-razorpay#187]
I have tried with automatic as well as manual installation. Also followed the video and each step but still getting below error : 

dyld: Library not loaded: @rpath/libswiftCore.dylib
  Referenced from: /Users/team/Library/Developer/CoreSimulator/Devices/BDD261DA-5D3D-4DAB-A13B-3C49B52DFE24/data/Containers/Bundle/Application/078AEA8D-988B-43DF-806D-20FF0B71468E/rzrPay.app/Frameworks/Razorpay.framework/Razorpay
  Reason: image not found

**Comment by pronav** [comment]
Which Xcode version is it, specifically? Latest version of SDK works only with Xcode 10.2, for 10.1 you can install v1.0.28

**Comment by ravifullestop** [comment]
I am using Xcode 10.1, Let me try again with Xcode 10.2 and I will let you know. Give me few hours to get back to you.

**Comment by ravifullestop** [comment]
Still getting below error : 

info ld: warning: Could not find auto-linked framework 'Razorpay'
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$__TtC8Razorpay8Razorpay", referenced from:

info       objc-class-ref in libRazorpayCheckout.a(RazorpayCheckout.o)

info ld: symbol(s) not found for architecture x86_64

info clang: error: linker command failed with exit code 1 (use -v to see invocation)

info 

error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening rzrPaayyy.xcodeproj

** BUILD FAILED **


The following commands produced analyzer issues:
	Analyze /Users/team/Documents/Training/reactnative/rzrPaayyy/node_modules/react-native/ReactCommon/jsi/jsi.cpp normal x86_64
	Analyze /Users/team/Documents/Training/reactnative/rzrPaayyy/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp normal x86_64
	Analyze /Users/team/Documents/Training/reactnative/rzrPaayyy/node_modules/react-native/React/Base/RCTModuleMethod.mm normal x86_64
(3 commands with analyzer issues)

The following build commands failed:
	Ld /Users/team/Documents/Training/reactnative/rzrPaayyy/ios/build/rzrPaayyy/Build/Products/Debug-iphonesimulator/rzrPaayyy.app/rzrPaayyy normal x86_64
(1 failure)
. Run CLI with --verbose flag for more details.

**Comment by pronav** [comment]
You've to add Razorpay.framework in embedded frameworks section of your project build settings.

**Comment by ravifullestop** [comment]
Yes it worked by manual installation only. No automatic installation working by running comment react-native link react-native-razorpay.

However below link is working : 

https://github.com/razorpay/react-native-razorpay/wiki/Manual-Installation
